src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -2618,6 +2618,13 @@ PeerImp::onMessage(std::shared_ptr<protocol::TMGetObjectByHash> const& m)
                         newObj.set_ledgerseq(obj.ledgerseq());
 
                     // VFALCO NOTE "seq" in the message is obsolete
+
+                    // Check if by adding this object, reply has reached its
+                    // limit
+                    if (reply.objects_size() == Tuning::hardMaxReplyNodes)
```

Comment:
```suggestion
                    if (reply.objects_size() >= Tuning::hardMaxReplyNodes)
```

This is probably more of a personal style choice, but I think `>=` is safer than `==` at almost no cost. For example, if the loop is changed in the future in a way that could somehow add more than one object in one pass of the loop, it could be very easy for the size to "skip over"  the exact match.